<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>التهيئة</string>
  </property>
  <widget class="QLineEdit" name="txtClassName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color:rgb(255, 255, 255);
    font: 75 8pt &quot;Motken K Sina&quot;;
    color:rgb(0, 0, 0);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>اسم الفصل</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string> ر ائد الفصل</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveClasses">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>حفــظ</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/true.png</normaloff>:/icons_qrc/true.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAddingClasses">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Motken K Sina</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>الغــــاء</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/exclamation-48.ico</normaloff>:/icons_qrc/exclamation-48.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="labAddClassRoom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

background-color:qlineargradient(spread:pad, x1:0.506975, y1:0.432, x2:0.99, y2:0.437227, stop:0.900498 rgba(4, 0, 62, 255), stop:1 rgba(255, 255, 255, 255));
color:rgb(255, 255, 255);
font: 15pt &quot;Motken K Sina&quot;;
</string>
   </property>
   <property name="text">
    <string>أضافة الصفوف الدراسية</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboMajorName">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>160</y>
     <width>231</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>40</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 10pt &quot;Motken K Sina&quot;;
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSkipClasses">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>تخطي</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/arrow-left.png</normaloff>:/icons_qrc/arrow-left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveClass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>التالـــي</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/arrowLeft.png</normaloff>:/icons_qrc/arrowLeft.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>labAddClassRoom</tabstop>
 </tabstops>
 <resources>
  <include location="icons_rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
